README.md

```diff
@@ -16,8 +16,52 @@ apt-get update && apt-get install build-essential -y
 DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install libgl1 libglib2.0-0 -y
 ```
 
+
+## Submission Model
+
+### Wi8Ai8KVi8 quantized BERT (qBERT)
+
+- Evaluation result
+
+    |    |         our result          | accuracy target |
+    |:--:|:---------------------------:|:---------------:|
+    | F1 | 91.05864237161231 (100.20%) | [90.874](https://github.com/mlcommons/inference/blob/e39003a9c4c89a2215db0ca57ad7a57b16f9a785/tools/submission/submission_checker.py#L1119C31-L1119C37)
+
+- To reproduce the result, you can run one of the following commands:
+
+    - 
+        ```
+        make qbert
+        ```
+
+    - 
+        ```
+        . scripts/build_qbert_env.sh
+        . scripts/eval_qbert.sh
+        ```
+        
+- Some parameters are configurable like below.
+
+    ```
+        export SCENARIO=Offline # SCENARIO is one of [Offline, SingleStream, MultiStream, Server]
+        export N_COUNT=10833   # N_COUNT is a number between [1, 10833]
+        export CALIBRATE=false # CALIBRATE is one of [false, true]
+        export N_CALIB=100 # N_CALIB is a number between [1, 100]
+        make qbert
```

Comment:
아래 부분은 수정이 필요할 것 같습니다. "to reproduce the result" 라는 내용이 포함되어 있으면 위 결과를 동일하게 재현할 수 있는 각종 상세가 주어져야 할 것 같습니다. 기본설정으로 주어져있다 하더라도 명시적으로 설정하는 방식의 커맨드를 아래 써주시면 좋을 것 같습니다. 

````suggestion
To reproduce the above accuracy result, please run as follow:

```
. scripts/build_qbert_env.sh  # You can skip this step if you already setup the environment.
make qbert
```

For other evaluations with different settigs, you can also change the following environment variables. Each configuration is defined as follows:
* `SCENARIO` is a MLPerf scenario to evaluate the accuracy. It can be one of Offline, SingleStream, MultiStream, or Server.
* `N_COUNT`: the number of data to evaluate the accuracy ([1, 10833])
* `CALIBRATE`: the calibration is involved in the evaluation step if true. (default: ???)
* `N_CALIB`: the number of data to calibrate the quantized model ([1, 100])
````